CMakeLists.txt

```diff
@@ -0,0 +1,110 @@
+cmake_minimum_required(VERSION 3.15)
```

Comment:
I think the minimum required CMake version should be aligned with https://github.com/stickies-v/py-bitcoinkernel/blob/0ba897a112a62d521b92c0e41b7c670cb8e4ddbe/depend/bitcoin/CMakeLists.txt#L10

Reply:
Thanks, you're right, bumped to 3.22 in https://github.com/stickies-v/py-bitcoinkernel/pull/20!